packages/terra-heading/tests/nightwatch/HeadingWeight.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import Heading from '../../lib/Heading';
+
+const HeadingWeight = () => (
+  <div>
+    <Heading level={1} weight={700}>Weight</Heading>
```

Comment:
Updated: https://github.com/cerner/terra-core/pull/470/commits/286b3bc3f2998714cd1aa971c4c7120e9689b622

Added the additional weight examples